### DISCUSSION TITLE
Prevent target="_blank"

### DISCUSSION THREAD

**Discussion by ValeryVerkhoturov** [tw93/Pake#810] [Q&A] [answered]
Is it possible to open href that has target="_blank" without opening browser?

**Comment by ValeryVerkhoturov** [comment, marked as answer]
I solved it with injector

```js
const observer = new MutationObserver(() => {
  removeTargetBlank();
});

function removeTargetBlank() {
  document.querySelectorAll('a[target="_blank"]').forEach(link => {
    link.removeAttribute('target');
  });
}

const config = {
  childList: true,
  subtree: true
};

observer.observe(document.body, config);

```

**Comment by asharca** [comment]
Hi! I saw your solution and tried to use it by saving it as a .js file and injecting it with Pake's --inject, but it didn’t work for me. Could you explain how you used this code? Thanks!

**Reply by asharca** [reply]
I solved it! I moved the function body outside of the DOMContentLoaded event listener, and now it works.

**Reply by ValeryVerkhoturov** [reply]
Thanks, I updated the solution
